source/PlayerInfo.cpp

```diff
@@ -3663,6 +3663,14 @@ void PlayerInfo::RegisterDerivedConditions()
 	visitedSystemProvider.SetGetFunction(visitedSystemFun);
 	visitedSystemProvider.SetHasFunction(visitedSystemFun);
 
+	auto &&pluginProvider = conditions.GetProviderPrefixed("installed plugin: ");
+	auto pluginFun = [](const string &name) -> bool
+	{
+		return Plugins::Get().Find(name.substr(strlen("installed plugin: ")));
```

Comment:
This returns all plugins the game has identified, but those will not all necessarily be loaded.
This distinction is important because, if a plugin is not loaded, it has no impact on the game.
Each `Plugin` in the `Set` returned by `Plugins::Get` is a struct, so you can just check its `enabled` member after `Find`ing it.

Reply:
So change that line to this? 
`return Plugins::Get().Find(name.substr(strlen("installed plugin: ")))->enabled;`